OrbitGl/PresetsDataView.cpp

```diff
@@ -83,8 +83,11 @@ std::string PresetsDataView::GetValue(int row, int column) {
 }
 
 std::string PresetsDataView::GetToolTip(int row, int /*column*/) {
-  const PresetFile& preset = *GetPreset(row);
-  return preset.file_name();
+  const std::shared_ptr<PresetFile>& preset = GetPreset(row);
+  return absl::StrCat(preset->file_name(),
+                      app_->GetPresetLoadState(preset).state == PresetLoadState::kNotLoadable
+                          ? "<br/><br/><i>None of the modules in the preset are loaded.</i>"
```

Comment:
```suggestion
                          ? "<br/><br/><i>None of the modules in the preset can be loaded.</i>"
```

Reply:
Changed accordingly. Thanks for the suggestions!